tpp-app/src/services/cycle/CycleService.js

```diff
@@ -237,10 +235,39 @@ const CycleService = {
       return getDaysDiffInclusive(startDate, date);
     }
 
-    return periodDays;
 
   },
 
+ /**
+    * Get the days since the last period has ended.
+    * @return {Promise} Resolves into 0 if user on their period, and an integer of the days they have been on their period otherwise
+    */
+  GETDaysSinceLastPeriodEnd: async function (){
+    //TODO: add this to the wiki
+    var curr = new Date()
+    var days = 0;
+
+    let calendar = await getCalendarByYear(curr.getFullYear());
+
+    let currSymptoms = await getSymptomsFromCalendar(calendar, curr.getDate(), curr.getMonth() + 1, curr.getFullYear());
```

Comment:
```suggestion
    let currSymptoms = await getSymptomsFromCalendar(calendar, curr.getDate(), curr.getMonth() + 1, curr.getFullYear());
```
    let currSymptoms = getSymptomsFromCalendar(calendar, curr.getDate(), curr.getMonth() + 1, curr.getFullYear());